File: 251110/살 수 있는 물건 2/things-able-to-buy-2.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim();
const N = Number(input);

let result = null;
if (N >= 3000) {
    result = 'book';
} else if (N >= 1000) {
    result = 'mask';
} else if (N >= 500) {
    result = 'pen';
} else {
    result = 'no';
}

console.log(result);

File: 260311/살 수 있는 물건 2/things-able-to-buy-2.py
import sys

input = sys.stdin.readline

N = int(input())
result = None

if N >= 3000:
    result = 'book'
elif N >= 1000:
    result = 'mask'
elif N >= 500:
    result = 'pen'
else:
    result = 'no'

print(result)